emily/handler/Cargo.toml

```diff
@@ -21,6 +21,7 @@ tracing-attributes.workspace = true
 tracing-subscriber.workspace = true
 utoipa = "4.2.3"
 warp = "0.3"
+warp_lambda.workspace = true
```

Comment:
TIL about `warp_lambda`. Is this just temporary or is the intention to use warp within the lambda functions?

It's not obvious to me what the benefit of doing this would be, because the pivot we discussed was just about running utoipa and API declarations/routing did not seem to be a pain point with the previous setup. And we shouldn't need warp to use utoipa.

However, if we could build `Emily` as a pure warp service and then isolate `warp_lambda` and all the lambda specific boilerplate to only the production builds - this would make it much simpler to run Emily locally. So that would be amazing.

Would love to hear your thoughts here.

Reply:
I think running it as a warp service is something for the future, within the suggestion that I pivot emily to `utoipa` was pivoting the whole thing to use standard Rust libraries. So far it's already a big improvement because there's much less bespoke serialization and deserialization.

I do hear what you're saying, I think it this moment my goal is to move quickly and get what we know will work working.

I'll say, using warp is already saving a LOT of time, I should have known when I started.